src/main/resources/standardTestCases/sub_workflow_hello_world.test

```diff
@@ -0,0 +1,15 @@
+name: sub_workflow_hello_world
+testFormat: workflowsuccess
+
+files {
+  wdl: sub_workflow_hello_world/sub_workflow_hello_world.wdl
+  imports: [
+    sub_workflow_hello_world/sub_workflow_hello_world_import.wdl
```

Comment:
Last time I tried yes it was - only for one layer of imports though (if your imported file contains an import too it won't work)